tests/admin/test_controller.py

```diff
@@ -1969,14 +1957,6 @@ def test_collections_post_edit(self):
         eq_("website_id", setting.key)
         eq_("1234", setting.value)
 
-        setting = collection.external_integration.setting("default_loan_period")
-        eq_("default_loan_period", setting.key)
-        eq_("14", setting.value)
```

Comment:
This could be changed to check that the library + integration setting was set.

Reply:
Can you tell me what the setup code would look like? I tried passing in a value with the library short name and nothing happened (which might either indicate a bug or that I did it wrong).

Reply:
I think you have it right, but this is testing with an Overdrive collection and Overdrive doesn't have that setting.

Reply:
Actually, I looked and the admin interface code for collections doesn't handle library settings yet - this is the first time we've had them for collections.

Reply:
The JS part should be fine as is, but SettingsController.collections will need to be updated.